src/web/components/PortalHeader/PortalHeader.tsx

```diff
@@ -70,20 +72,22 @@ export function PortalHeader({
               )}
             </Avatar>
           </div>
-          <DropdownMenuSeparator className='separator' />
-          {routes.map((route) => {
-            return (
-              <DropdownMenuItem
-                key={route.path}
-                className='dropdown-menu-item'
-                onClick={handleSelect}
-              >
-                <Link to={route.path} className='link'>
-                  {route.description}
-                </Link>
-              </DropdownMenuItem>
-            );
-          })}
+          {LoggedInUser?.user && <DropdownMenuSeparator className='separator' />}
```

Comment:
nit: I think this is slightly more readable:
```
{LoggedInUser?.user && (
  <>
    <DropdownMenuSeparator className='separator' />
    {routes.map((route) => {
      return (
        <DropdownMenuItem
          key={route.path}
          className='dropdown-menu-item'
          onClick={handleSelect}
        >
          <Link to={route.path} className='link'>
            {route.description}
          </Link>
        </DropdownMenuItem>
      );
    })}
  </>
)}
```

Reply:
yeah you're right.  I changed it.